src/open_inwoner/scss/components/Form/ButtonRadio.scss

```diff
@@ -4,6 +4,7 @@
 
     &:checked ~ .button-radio__button {
       color: var(--color-primary);
+      cursor: default;
```

Comment:
It seems weird that we do not want the pointer cursor. What is the reason for this?

Reply:
Before the button is pressed the cursor is a pointer. But after the button is pressed I decided to move it back to arrow, since the second pressing won't do anything. I can make it always a pointer

Reply:
Thanks for the explanation. You do not have to change anything. This is indeed better.